ApiBundle/Controller/DeletedController.php

```diff
@@ -27,7 +26,7 @@ class DeletedController extends BaseController
      */
     public function listAction()
     {
-        /** @var Collection $nodes */
+        /** @var Array $nodes */
```

Comment:
you should put `array` instead of `Array` and put the comment line just above the $nodes line definition